### ISSUE TITLE
Subscripts affect line height

### ISSUE THREAD

**Issue by tsktsktsk64** [cpitclaudel/company-coq#194]
Hi there. When have `company-coq-features/smart-subscripts` turned on, the difference between the line height of the lines with subscripts and without subscripts is pretty big, as shown in the README and my own screenshots. Is there a way to use smart subscripts with consistent line height throughout the whole file? Perhaps by trimming the bottom part of the subscripts?

I tried to change the `Height` property of the `Company Coq Features/Smart Subscripts Face` to a smaller value, and change the `raise` value in `company-coq-features/smart-subscripts--subscript-spec` in `company-coq.el` to 0. But if done this way, the baseline of the normal text and subscripts are the same, making them not distinguishable from each other, which is still not ideal.

The default settings:
![image](https://user-images.githubusercontent.com/10580362/37559866-4ca57120-2a68-11e8-8dae-760355122c11.png)

The settings I tried:
![image](https://user-images.githubusercontent.com/10580362/37559871-70e1df74-2a68-11e8-8584-26a71c3d9509.png)

**Comment by cpitclaudel** [comment]
Yup, I'm aware of the issue; but unfortunately I don't know a solution — what you did is essentially what I was going to offer.
